zuul/tenants.yaml

```diff
@@ -39,6 +39,7 @@
           - ansible-collections/cloud.common
           - ansible-collections/community.asa
           - ansible-collections/community.aws
+          - ansible-collections/community.ciscosmb
```

Comment:
will you be gating these project with zuul? EG: zuul merging, or just want access to publish jobs?

Reply:
if just publish jobs, please move to line 106 an below

Reply:
Just publish

Reply:
Done